src/python/py_image_generation_pipelines.cpp

```diff
@@ -275,7 +275,7 @@ void init_image_generation_pipelines(py::module_& m) {
             (text2image_generate_docstring + std::string(" \n ")).c_str()
         );
 
-    auto image_generation_scheduler = py::class_<ov::genai::Scheduler>(m, "Scheduler", "Scheduler for image generation pipelines.")
+    auto image_generation_scheduler = py::class_<ov::genai::Scheduler, std::shared_ptr<ov::genai::Scheduler>>(m, "Scheduler", "Scheduler for image generation pipelines.")
         .def(py::init<>())
```

Comment:
@popovaan 
can we drop this ctor?

Reply:
Yes, made a PR to drop it: https://github.com/openvinotoolkit/openvino.genai/pull/1136